src/app/games/epaminondas/tests/EpaminondasRules.spec.ts

```diff
@@ -281,29 +298,15 @@ describe('EpaminondasRules:', () => {
                 [X, _, _, _, _, _, _, _, _, _, _, _, _, _],
                 [_, _, _, _, _, _, _, _, _, _, _, _, _, _],
             ];
-            const state: EpaminondasState = new EpaminondasState(board, 1);
+            const state: EpaminondasState = new EpaminondasState(board, 2);
             const move: EpaminondasMove = new EpaminondasMove(0, 9, 1, 1, Direction.DOWN);
-            const expectedState: EpaminondasState = new EpaminondasState(expectedBoard, 2);
-            RulesUtils.expectMoveSuccess(rules, state, move, expectedState);
-            const node: EpaminondasNode = new EpaminondasNode(expectedState, MGPOptional.empty(), MGPOptional.of(move));
+            const node: EpaminondasNode = new EpaminondasNode(state, MGPOptional.empty(), MGPOptional.of(move));
+            // Then it should be a victory for 0
             RulesUtils.expectToBeVictoryFor(rules, node, Player.ZERO, minimaxes);
         });
-        it('Should declare second player winner if his pawn survive one turn on first line', () => {
+        it('should declare second player winner if his pawn survive one turn on first line', () => {
```

Comment:
its survival is actually not the rule that make its owner the winner, it's actually that, at the beginning of your turn you (would?) have more piece on the opponent last line the the opponent does on yours !
Could be nice to make sure we emphased this correctly, especially not misleading user in the tutorial for one of the best game on MGP

Reply:
c'est clair dans le tuto non ? « Après plusieurs déplacements, si au début de son tour de jeu, un joueur a plus de pièces sur la ligne de départ de l'adversaire que l'adversaire n'en a sur la ligne de départ du joueur, ce joueur gagne. »

Pour ce test par contre, c'est bien ça qu'il teste : que si ce pion survit, le joueur gagne. Ou tu veux une formulation plus générique (mais alors comment distinguer un test de victoire d'un autre ?), ou plus précise (should declare second player winner if their pawn survives until next turn on the last line and the opponent has no pawn on the last line) ?

Reply:
c'est toujours mieux que le it décrive une VRAI attente en terme de behavior, pas trop (si possible) le "ici là, d'une certaines façon, c'est ça qui doit se passer" mais ouais si on fait la vraie règle ça va être a-rallongis :D

Reply:
héhé oui ! Du coup ?